### PR TITLE
py-telethon: Submission

### PR DESCRIPTION
**Pull request by essandess**
###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS 12.4 21F79 x86_64
Xcode 13.4 13F17a

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [x] checked your Portfile with `port lint --nitpick`?
- [ ] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

File: python/py-telethon/Portfile [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+description         Full-featured Telegram client library.
+long_description    Telethon is an asyncio Python 3 MTProto library to \
+                    interact with Telegram's API as a user or through \
+                    a bot account (bot API alternative).
+
+homepage            https://github.com/LonamiWebs/Telethon
+
+python.versions     37 38 39 310
+
+distname            [string totitle ${distname}]
```
Comment (reneeotten) [inline python/py-telethon/Portfile:21]: instead, please set `python.rootname Telethon` - without the Tcl magic please, just write it out for us mortals to understand ;) That will also make sure that the `livecheck.name` is automatically set correctly.

File: python/py-telethon/Portfile [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+maintainers         nomaintainer
+
+description         Full-featured Telegram client library.
+long_description    Telethon is an asyncio Python 3 MTProto library to \
+                    interact with Telegram's API as a user or through \
+                    a bot account (bot API alternative).
+
+homepage            https://github.com/LonamiWebs/Telethon
+
+python.versions     37 38 39 310
```
Comment (reneeotten) [inline python/py-telethon/Portfile:19]: do you need all the older ones as well or can we just stick to `310`?

File: python/py-telethon/Portfile [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+if {${name} ne ${subport}} {
+    depends_lib-append  \
+                    port:py${python.version}-setuptools
+
+    depends_lib-append  \
+                    port:py${python.version}-markdown-it-py \
+                    port:py${python.version}-pyaes \
+                    port:py${python.version}-rsa
+
+    livecheck.type  none
```
Comment (reneeotten) [inline python/py-telethon/Portfile:36]: this is set automatically by the `python` PG, so the line can be removed

File: python/py-telethon/Portfile [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+python.versions     37 38 39 310
+
+distname            [string totitle ${distname}]
+
+checksums           rmd160  7ebef3569ba02fcdb5886efba4b5d8183ad6b4a7 \
+                    sha256  818cb61281ed3f75ba4da9b68cb69486bed9474d2db4e0aa16e482053117452c \
+                    size    464470
+
+if {${name} ne ${subport}} {
+    depends_lib-append  \
```
Comment (reneeotten) [inline python/py-telethon/Portfile:28]: this should likely be `depends_build-append`?

**Review by reneeotten — CHANGES_REQUESTED** [review]

File: python/py-telethon/Portfile (hunk 1 above)
Comment (essandess) [reply python/py-telethon/Portfile]: Thanks done.

File: python/py-telethon/Portfile (hunk 2 above)
Comment (essandess) [reply python/py-telethon/Portfile]: I kept `38 39 310`.

File: python/py-telethon/Portfile (hunk 4 above)
Comment (essandess) [reply python/py-telethon/Portfile]: Thanks done.

File: python/py-telethon/Portfile (hunk 3 above)
Comment (essandess) [reply python/py-telethon/Portfile]: Thanks done.

**Pull request merged**